model/project_ref.go

```diff
@@ -284,7 +284,11 @@ func (p *ProjectRef) AddToRepoScope(user *user.DBUser) error {
 				Enabled: true,
 			}}
 			// creates scope and give user admin access to repo
-			return errors.Wrapf(repoRef.Add(user), "problem adding new repo ref for '%s/%s'", p.Owner, p.Repo)
+			err = repoRef.Add(user)
+			if err != nil {
```

Comment:
nit; you can put this on one line since you don't need the error outside of the if statement (`if err = repoRef.Add(user); err != nil`)